code/game/objects/items/extinguisher.dm

```diff
@@ -60,6 +60,7 @@
 	tanktype = /obj/structure/reagent_dispensers/foamtank
 	sprite_name = "foam_extinguisher"
 	precision = TRUE
+	w_class = WEIGHT_CLASS_NORMAL
```

Comment:
No.

Reply:
fire extinguishers are the topic of today's balance discussion 😔